snuba/datasets/configuration/generic_metrics/entities/distributions.yaml

```diff
@@ -110,7 +110,7 @@ translation_mappers:
         column_to_map: value
         from_name: minIf
         to_name: minMergeIf
-        aggr_col_name: minIf
+        aggr_col_name: min
```

Comment:
This caused API errors in `make test-snuba` in Sentry:

```
clickhouse_driver.errors.ServerException: Code: 47.
DB::Exception: Missing columns: 'minIf' while processing query: 'SELECT maxMergeIf(max, (metric_id AS _snuba_metric_id) = 9223372036854775909) AS _snuba_max_transaction_duration, minMergeIf(minIf, _snuba_metric_id = 9223372036854775909) AS _snuba_min_transaction_duration FROM generic_metric_distributions_aggregated_local WHERE (granularity = 3) AND ((timestamp AS _snuba_timestamp) >= toDateTime('2022-07-14T00:00:00', 'Universal')) AND (_snuba_timestamp < toDateTime('2022-10-12T22:23:44', 'Universal')) AND ((project_id AS _snuba_project_id) IN [4550720995983361]) AND ((org_id AS _snuba_org_id) = 4550720995983360) AND (_snuba_metric_id IN [9223372036854775909]) LIMIT 0, 1', required columns: 'granularity' 'timestamp' 'project_id' 'org_id' 'max' 'metric_id' 'minIf', source columns: '_raw_tags_hash' '_indexed_tags_hash' 'count' 'min' 'percentiles' 'tags.raw_value' 'histogram_buckets' 'max' 'use_case_id' 'granularity' 'tags.key' 'avg' 'tags.indexed_value' 'sum' 'org_id' 'metric_id' 'project_id' 'timestamp' 'retention_days'. Stack trace:

0. Poco::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x105351b0 in /usr/bin/clickhouse
1. DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x8f4172d in /usr/bin/clickhouse
2. ? @ 0xd4e9f18 in /usr/bin/clickhouse
3. DB::SyntaxAnalyzer::analyzeSelect(std::__1::shared_ptr<DB::IAST>&, DB::SyntaxAnalyzerResult&&, DB::SelectQueryOptions const&, std::__1::vector<DB::TableWithColumnNamesAndTypes, std::__1::allocator<DB::TableWithColumnNamesAndTypes> > const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&) const @ 0xd4e632f in /usr/bin/clickhouse
4. ? @ 0xd139ce5 in /usr/bin/clickhouse
5. DB::InterpreterSelectQuery::InterpreterSelectQuery(std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, std::__1::shared_ptr<DB::IBlockInputStream> const&, std::__1::optional<DB::Pipe>, std::__1::shared_ptr<DB::IStorage> const&, DB::SelectQueryOptions const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&) @ 0xd13b05e in /usr/bin/clickhouse
6. DB::InterpreterSelectQuery::InterpreterSelectQuery(std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, DB::SelectQueryOptions const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&) @ 0xd13c619 in /usr/bin/clickhouse
7. DB::InterpreterSelectWithUnionQuery::InterpreterSelectWithUnionQuery(std::__1::shared_ptr<DB::IAST> const&, DB::Context const&, DB::SelectQueryOptions const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&) @ 0xd341686 in /usr/bin/clickhouse
8. DB::InterpreterFactory::get(std::__1::shared_ptr<DB::IAST>&, DB::Context&, DB::QueryProcessingStage::Enum) @ 0xd0909b4 in /usr/bin/clickhouse
9. ? @ 0xd550655 in /usr/bin/clickhouse
10. DB::executeQuery(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, DB::Context&, bool, DB::QueryProcessingStage::Enum, bool, bool) @ 0xd553441 in /usr/bin/clickhouse
11. DB::TCPHandler::runImpl() @ 0x9024489 in /usr/bin/clickhouse
12. DB::TCPHandler::run() @ 0x9025470 in /usr/bin/clickhouse
13. Poco::Net::TCPServerConnection::start() @ 0xe3ac69b in /usr/bin/clickhouse
14. Poco::Net::TCPServerDispatcher::run() @ 0xe3acb1d in /usr/bin/clickhouse
15. Poco::PooledThread::run() @ 0x105c3317 in /usr/bin/clickhouse
16. Poco::ThreadImpl::runnableEntry(void*) @ 0x105bf11c in /usr/bin/clickhouse
17. ? @ 0x105c0abd in /usr/bin/clickhouse
18. start_thread @ 0x76db in /lib/x86_64-linux-gnu/libpthread-2.27.so
19. __clone @ 0x12188f in /lib/x86_64-linux-gnu/libc-2.27.so
```

I had incorrectly transcribed from the python entity definition but we never caught this in test because the setting I'm removing here disabled it